web/packages/teleterm/src/ui/services/workspacesService/accessRequestsService.ts

```diff
@@ -231,10 +237,16 @@ export type ResourceRequest =
       kind: 'app';
       resource: {
         uri: AppUri;
+        samlApp: boolean;
```

Comment:
```suggestion
        isSamlApp: boolean;
```

Reply:
Unfortunately, if we change the property name, the [`App` type](https://github.com/gravitational/teleport/blob/39adc9883733a7a954f69392ebd11b8d07649b7a/gen/proto/ts/teleport/lib/teleterm/v1/app_pb.ts#L109) won't be assignable to this type.

Reply:
Yeah cannot change that here. This is the same field naming as used in the App type proto and also the App type defined in the Web codebase.